README.md

```diff
@@ -136,7 +137,8 @@ sessions:
 - `host: string` (optional, default: hostname of localhost): Target host of the `tmux` session.
 - `user: string` (optional, default: robot): Target user of the `tmux` session.
 - `wait_for_core: bool` (optional, default: true): Starts the session only after `roscore` is available.
-- `prio: int` (optional, default: 10): Priority of the session. Sessions with the same priority start concurrently. Smaller numbers have higher priority.
+- `prio: int` (optional, default: 10): Priority of starting the session. Sessions with the same priority start concurrently. Smaller numbers have higher priority.
+- `stop_prio: int` (optional, default: `prio`): Priority of stopping the session. Sessions with the same priority stop concurrently. Higher numbers have higher priority.
```

Comment:
I would have suggested to go for `prio_start` and `prio_stop`, but I see the issue with backwards compatibility
I would still go for `prio_stop` instead of `stop_prio`....

I can do the change

Reply:
Change is implemented.